.github/workflows/continuous-integration.yml

```diff
@@ -82,18 +82,24 @@ jobs:
         env:
           YARN_CACHE_FOLDER: ~/.cache/yarn
 
-      - name: Run ESLint
+      - name: Run ESLint (Report)
```

Comment:
Could we run ESLint only once? If the annotations can be created from console logs, theoretically you could generate the JSON while console logging the annotations. But I'm also not sure of the value of having this extra ESLint report if we're already providing the file names and line numbers in the summary, although the report does provide a nicer format.

What are your thoughts for the best developer experience? I'd mainly want to avoid running the linter twice.

Reply:
I think we should just avoid running lint twice and remove the `generate ESlint report` as the information is now being displayed immediately. Seems redundant and we can shave off a few mins (~3) by not having it run twice

Reply:
Alternatively, maybe you could do `--format json` and then for the annotation, parse the JSON output and `console.log`.

Reply:
unfortunately I have to pass in the JS file I created for formatting as when I do `--format json` it does not provide the format for GHA to have detailed output